ts/client/src/client.ts

```diff
@@ -1855,11 +1855,11 @@ export class MangoClient {
     group: Group,
     mangoAccount: MangoAccount,
     externalMarketPk: PublicKey,
-    side: Serum3Side,
+    side: Serum3Side | any,
```

Comment:
<img width="997" alt="image" src="https://github.com/blockworks-foundation/mango-v4/assets/89031858/de1ed088-351a-4fd6-9c69-69ae2bb54288">

So basically we would have to move away from current to globals, see both current and global example below
![image](https://github.com/blockworks-foundation/mango-v4/assets/89031858/48a5ee81-4e1d-466e-9bae-0fb34b54ab5e)

Reply:
I think my current workaround is better, we keep code as is, but have a `| any` so the caller knows the main type he should use, and code compiles

Reply:
Alternatively we could disable compile checking completely for these lines, or do you have other suggestions?